caikit/runtime/utils/servicer_util.py

```diff
@@ -292,16 +292,15 @@ def validate_data_model(
                     )
                     continue
 
-                field_message_type = input_proto_msg.fields_by_name[
-                    field.name
-                ].message_type
+                field_descriptor = input_proto_msg.fields_by_name[field.name]
```

Comment:
I think this can be done with a smaller delta by just adding `if field_message_type` to the beginning of the `if`